os/linux/gbmbufferhandler.cpp

```diff
@@ -85,9 +85,9 @@ bool GbmBufferHandler::CreateBuffer(uint32_t w, uint32_t h, int format,
   temp->import_data.height = gbm_bo_get_height(bo);
   temp->import_data.format = gbm_bo_get_format(bo);
 #if USE_MINIGBM
-  temp->import_data.fds[0] = gbm_bo_get_plane_fd(bo, 0);
   size_t total_planes = gbm_bo_get_num_planes(bo);
   for (size_t i = 0; i < total_planes; i++) {
+    temp->import_data.fds[i] = gbm_bo_get_plane_fd(bo, i);
```

Comment:
int drv_prime_bo_import(struct bo *bo, struct drv_import_fd_data *data)
{
  int ret;
  size_t plane;
  struct drm_prime_handle prime_handle;

  for (plane = 0; plane < bo->num_planes; plane++) {
    memset(&prime_handle, 0, sizeof(prime_handle));
    prime_handle.fd = data->fds[plane];

    ret = drmIoctl(bo->drv->fd, DRM_IOCTL_PRIME_FD_TO_HANDLE, &prime_handle);



so plane fds can't be set to zero